src/docfx/lib/git/FileCommitProvider.cs

```diff
@@ -238,9 +239,12 @@ private unsafe (List<Commit>, Dictionary<long, Commit>) LoadCommits(string commi
                 if (error != 0 /* GIT_ENOTFOUND */)
                 {
                     git_revwalk_free(walk);
-                    throw new NotSupportedException($"Error computing git log '{error}' for '{_repoPath}'");
+
+                    Log.Write($"Load git commit failed: {error} {lastCommitId}");
+                    throw Errors.GitCloneIncomplete(_repoPath).ToException();
                 }
 
+                lastCommitId = commitId;
```

Comment:
what is this assignment for since it is not used any further?

Reply:
used in `Log.Write`

Reply:
`Log.Write` is called before this assignment

Reply:
I see now